tests/integration/operator/operator_test.go

```diff
@@ -0,0 +1,179 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR Condition OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integrationoperator
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	g "maistra.io/istio-operator/pkg/util/tests/ginkgo"
+	"maistra.io/istio-operator/pkg/util/tests/kubectl"
+	resourcecondition "maistra.io/istio-operator/pkg/util/tests/types"
+)
+
+var _ = Describe("Operator", Ordered, func() {
+	SetDefaultEventuallyTimeout(120 * time.Second)
+	SetDefaultEventuallyPollingInterval(time.Second)
+	var (
+		resourceAvailable = resourcecondition.Condition{
+			Type:   "Available",
+			Status: "True",
+		}
+		resourceReconcilied = resourcecondition.Condition{
+			Type:   "Reconciled",
+			Status: "True",
+		}
+
+		resourceReady = resourcecondition.Condition{
+			Type:   "Ready",
+			Status: "True",
+		}
+	)
+
+	Describe("installation", func() {
+		// TODO: we  need to support testing both types of deployment for the operator, helm and olm via subscription.
+		// Discuss with the team if we should add a flag to the test to enable the olm deployment and don't do that deployment in different step
+		When("default helm manifest are applied", func() {
+			BeforeAll(func() {
+				if skipDeploy == "true" {
+					Skip("Skipping the deployment of the operator and the tests")
+				}
+				g.Success("Deploying Operator using default helm charts located in /chart folder")
```

Comment:
I think we should use a dot import for our own ginkgo helper functions like this one.
```suggestion
				Success("Deploying Operator using default helm charts located in /chart folder")
```

Reply:
Done